plugins/GSdx/Renderers/DX11/GSTextureFX11.cpp

```diff
@@ -166,19 +166,22 @@ void GSDevice11::SetupGS(GSSelector sel, const GSConstantBuffer* cb)
 		}
 		else
 		{
+			int id = 0;
 			std::string str[4];
 
-			str[0] = format("%d", sel.iip);
-			str[1] = format("%d", sel.prim);
-			str[2] = format("%d", sel.point);
-			str[3] = format("%d", sel.line);
+			str[id++] = format("%d", sel.iip);
+			str[id++] = format("%d", sel.prim);
+			str[id++] = format("%d", sel.point);
+			str[id++] = format("%d", sel.line);
```

Comment:
Honestly I think it's better the old way as this makes me do math in my head in order to read.

Reply:
But the previous way forced me to count and I'm bad at it, so it was error prone.